airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -87,7 +87,7 @@ def Popen(self, cmd, **kwargs):
         :param kwargs: extra arguments to Popen (see subprocess.Popen)
         :return: handle to subprocess
         """
-        self.log.info("Executing command: %s", ' '.join(cmd))
+        self.log.info("Executing command: {}".format(' '.join(self.cmd_mask_password(cmd))))
```

Comment:
The function to mask the password was created but never called on. We should mask it in the `cmd` when we're logging it.

Reply:
I've written the function of masking the password earlier, but this should have been tested. Thanks